src/main/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/InterchangeHeader.java

```diff
@@ -56,4 +60,13 @@ public void preValidate() throws EdifactValidationException {
             throw new EdifactValidationException(getKey() + ": Attribute recipient is required");
         }
     }
+
+    public static InterchangeHeader fromString(String edifactString) {
+        if(!edifactString.startsWith(InterchangeHeader.KEY)){
+            throw new IllegalArgumentException("Can't create " + InterchangeHeader.class.getSimpleName() + " from " + edifactString);
+        }
+        String[] split = edifactString.split("\\+");
+        ZonedDateTime translationTime = ZonedDateTime.parse(split[4], DateTimeFormatter.ofPattern("yyMMdd:HHmm").withZone(ZoneId.of("Europe/London")));
```

Comment:
There will be a const for zone in my PR https://github.com/nhsconnect/integration-adaptor-nhais/pull/51/files#diff-b06dd46870f0ae485dc157d0f2b7ad1cR11

Reply:
I can mix that up in next PR (this one is covering only half of story)